py-pkg/setup.py

```diff
@@ -0,0 +1,13 @@
+from setuptools import setup
+
```

Comment:
totally fine with me if we push `description`, `url`, `author`, etc. in this file off to later PRs. But I do think it would be worth collecting all the dependencies here in `install_requires`. So that @bburns632 and I can see how much stuff is being imported. Could you do that?